apps/modernization-ui/src/apps/page-builder/page/management/usePageManagement.tsx

```diff
@@ -1,14 +1,20 @@
-import { createContext, ReactNode, useContext, useMemo, useState } from 'react';
+import { createContext, ReactNode, useContext, useEffect, useMemo, useState } from 'react';
 import { PagesResponse, PagesTab } from 'apps/page-builder/generated';
 
 type Selected = PagesTab | undefined;
-
+type AlertMessage = {
+    type?: 'error' | 'success' | 'warning';
+    message?: string | ReactNode;
+    expiration?: number | undefined;
+};
 type Interactions = {
     page: PagesResponse;
     selected?: Selected;
     fetch: (page: number) => void;
     refresh: () => void;
     select: (tab: PagesTab) => void;
+    alert?: AlertMessage;
+    setAlert: (alert: AlertMessage) => void;
```

Comment:
Question/Issue:
Part1:  All other errors trigger using the `useAlert` hook and are displayed in a toast. If alerts should now be handled differently, then i think this should be its own story to update all of the alerts to behave in a consistent manner

Part2: I don't think I'm sold on tacking the alert functionality to the `usePageManagement` hook. I'm not against it, just need to think it over a bit. The functionality to trigger an alert and have it displayed in a particular place is something that i think we will be using elsewhere. That is why i lean towards having the alert be a separate hook.

Reply:
These are for EditPage level alerts which seem to be where a lot of alerts are going to be displayed, above the header menus. I expect there to be a lot of child components that will need to be drilled down into to set alerts at this root level. Is there another way to keep things nice and tidy without drilling props all over the place?

Reply:
It makes sense to keep it in a page context as they are page-level alerts

Reply:
Ideally the `useAlert` hook would be used for all alert handling.  The problem is that the `AlertProvider` is also dictating how alerts are displayed.  Have the alert banners for this page been created yet?  Is it fine to use the existing alert hook and have it display incorrectly until the display is separated from the provider?

Reply:
> Ideally the `useAlert` hook would be used for all alert handling. The problem is that the `AlertProvider` is also dictating how alerts are displayed. Have the alert banners for this page been created yet? Is it fine to use the existing alert hook and have it display incorrectly until the display is separated from the provider?

Most alerts on this page are displayed using `useAlert`. There are a few modals that have their own internal `AlertBanner` that do not utilize `useAlert` for displaying messages. 

I have reached out to the design team to attempt to get clarification and make my push for consistency.

Reply:
@adamloup-enquizit @mpeels 
After confirming with Jared, I changed the Alert Banner to the toast notification--these designs will be updated eventually.